snoop/data/management/commands/filestats.py

```diff
@@ -0,0 +1,181 @@
+            else:
+                ext_dict[ext]['size'] += truncate_size(int(size))
+                ext_dict[ext]['mtype'].add(mime)
+    sorted_ext_dict = sorted(ext_dict.items(), key=lambda x: x[1]['size'], reverse=True)
+    return dict(sorted_ext_dict)
+
+
+def get_description(col, mime_type, *extension):
+    """Return the magic description for a given mime-type.
+
+    Args:
+        col: Collection on which the query is executed.
+        mime_type: Mime-Type for which the descriptions is returned.
+        *extension: Optional file-extension as string to limit the search to this
+            extension.
+    """
+
+    collection = collections.ALL[col]
+    with collection.set_current():
+        try:
+            if extension:
+                queryset = models.File.objects\
+                    .annotate(str_name=RawSQL("encode(name_bytes::bytea, 'escape')::text", ()))\
+                    .filter(blob__mime_type=mime_type, str_name__endswith=extension[0])\
+                    .values("blob__magic")[0]
+            else:
+                extension = [""]
+                queryset = models.File.objects\
+                    .annotate(str_name=RawSQL("encode(name_bytes::bytea, 'escape')::text", ()))\
+                    .filter(blob__mime_type=mime_type)\
+                    .values("blob__magic")[0]
+        except IndexError:
+            return None
+    return queryset['blob__magic']
+
+
+class Command(BaseCommand):
+    """Print the statistics for mimetypes or file-extendsion"""
+    help = "Display filetype stats."
+
+    def add_arguments(self, parser):
+        """Arguments to show only unsupported types, include magic descriptions,
+        include full magic descriptions and for choosing specific collections"""
+
+        parser.add_argument(
+            '--unsupported',
+            action='store_true',
+            help='exclude supported filetypes')
+
+        parser.add_argument(
+            '--descriptions',
+            action='store_true',
+            help='print MIME-type descriptions')
+
+        parser.add_argument(
+            '--full_descriptions',
+            action='store_true',
+            help='print full MIME-type descriptions')
+
+        parser.add_argument(
+            '--collections',
+            nargs='+',
+            type=str,
+            help='specify collections')
+
+    def handle(self, **options):
+        """Prints out the Top 100 (or so) mime-types and file extensions:
+
+        Results are sorted by total file size usage.
+        """
+        collection_args = list(collections.ALL.keys())
+        supported = True
+        if options['unsupported']:
+            supported = False
+        if options['collections']:
+            collection_args = options['collections']
+
+        print('Top Mime Types by size')
+        print('-----------------------')
+        for k, v in get_top_mime_types(collections_list=collection_args, print_supported=supported).items():
```

Comment:
here do `size =  v["size"] / ( 2 ** 20 ))` and display this float with 2 digits after the `.` and the `MB` unit after

i'm getting dizzy from all the long numbers, so please format the float to also include the `,` every 3 digits so we easily see when we pass into the GB range